Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyBoolComboBoxCtrl.cpp

```diff
@@ -71,14 +77,42 @@ namespace AzToolsFramework
     }
     QWidget* PropertyBoolComboBoxCtrl::GetLastInTabOrder()
     {
-        return m_pComboBox;
+        return m_editButton;
     }
 
     void PropertyBoolComboBoxCtrl::UpdateTabOrder()
     {
-        // There's only one QT widget on this property.
+        setTabOrder(GetFirstInTabOrder(), GetLastInTabOrder());
+    }
+
+    QComboBox* PropertyBoolComboBoxCtrl::GetComboBox()
+    {
+        return m_pComboBox;
+    }
+
+    QToolButton* PropertyBoolComboBoxCtrl::GetEditButton()
+    {
+        return m_editButton;
     }
 
+    void PropertyBoolComboBoxCtrl::SetEditButtonCallBack(AZ::Edit::AttributeFunction<int(int)>* function)
+    {
+        m_editButtonCallback = function;
+    }
+
+    void PropertyBoolComboBoxCtrl::OnEditButtonClicked()
+    {
+        if (m_editButtonCallback)
+        {
+            // intValue refers to the index value the comboBox will be set to on the return of the invoked callback
+            auto intValue = m_editButtonCallback->Invoke(nullptr, m_pComboBox->currentIndex());
+
+            if (intValue == 0 || intValue == 1)
```

Comment:
question: Is it possible to use an actual `bool` instead of `int` for this? It feels like it'd simplify things if possible